2020-02-Spring/2_linked_lists/README.md

```diff
@@ -45,22 +45,43 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. Remove Zero Sum Consecutive Nodes from Linked List
 
-Source: TODO :bug:
+Source: https://leetcode.com/problems/remove-zero-sum-consecutive-nodes-from-linked-list/
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given the head of a linked list, we repeatedly delete consecutive sequences of nodes that sum to 0 until there are no such sequences.
 
-#### Example Input
+After doing so, return the head of the final linked list.  You may return any such answer.
```

Comment:
Clarify what "any such answer" means

Reply:
I agree. Clarify what this means, or remove the second sentence completely.

If you are referring to a `nullptr` head, I wouldn't try to drop hints about that. The participants should be asking clarifying questions about that if it doesn't make sense.

```suggestion
After doing so, return the head of the final linked list.
```